src/Tempest/Cache/src/CacheConfig.php

```diff
@@ -6,19 +6,24 @@
 
 use Psr\Cache\CacheItemPoolInterface;
 use Symfony\Component\Cache\Adapter\FilesystemAdapter;
+use function Tempest\env;
 
 final class CacheConfig
 {
     /** @var class-string<\Tempest\Cache\Cache>[] */
     public array $caches = [];
 
+    public bool $enabled;
```

Comment:
This was kind of a quick fix to be able to disable cache altogether via env. I made an issue to follow up on it: https://github.com/tempestphp/tempest-framework/issues/561